pipe/pipe.py

```diff
@@ -142,6 +155,30 @@ def __cdk(self):
         if not status:
             return Exception('npm install: ' + str(err))     
 
+        # NPM Linting check
+        if (self.check_lint and self.cmd_npm_lint) or self.check_lint_override:
+            try:
+                # CHECK_LINT_CMD env value shall override the values from static script
+                check_linting_cmd = self.check_lint_override or self.cmd_npm_lint 
+                self.log_info("check linting [{}] => {}".format(self.working_dir,check_linting_cmd))
+                status, err = self.__scriptRunner(working_dir, [check_linting_cmd])
+                if not status:
+                    return Exception('check linting: ' + str(err)) 
+            except Exception as exception:
+                return Exception('linting check failed: ' + str(exception))# NPM Format check
```

Comment:
Is this comment meant to be at the eol?

Reply:
No it should not :D, thanks for spotting it